tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -0,0 +1,132 @@
+    Then the HTTP status code should be "207"
+    And the search result of user "Brian" should contain only these files:
+      | file1.txt |
+      | file2.txt |
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+
+
+  Scenario Outline: as a sharee search a file tagged by sharee using tag
+    Given using <dav-path-version> DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
+    And user "Alice" has created folder "uploadFolder"
+    And user "Alice" has uploaded file with content "hello world" to "uploadFolder/file1.txt"
+    And user "Alice" has uploaded file with content "Namaste nepal" to "uploadFolder/file2.txt"
+    And user "Alice" has uploaded file with content "hello nepal" to "uploadFolder/file3.txt"
+    And user "Alice" has shared folder "/uploadFolder" with user "Brian"
+    And user "Brian" has accepted share "/uploadFolder" offered by user "Alice"
+    And user "Brian" has created the following tags for file "uploadFolder/file1.txt" of the space "Shares":
+      | tag1 |
+    And user "Brian" has created the following tags for file "uploadFolder/file2.txt" of the space "Shares":
+      | tag1 |
+    When user "Brian" searches for "Tags:tag1" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result of user "Brian" should contain only these files:
+      | file1.txt |
+      | file2.txt |
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+
+
+  Scenario Outline: search files using a deleted tag
+    Given using <dav-path-version> DAV path
+    And user "Alice" has uploaded file with content "hello world" to "file1.txt"
+    And user "Alice" has uploaded file with content "Namaste nepal" to "file2.txt"
+    And user "Alice" has created the following tags for file "file1.txt" of the space "Personal":
+      | tag1 |
+    When user "Alice" removes the following tags for file "file1.txt" of space "Personal":
+      | tag1 |
+    When user "Alice" searches for "Tags:tag1" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result should contain "0" entries
+    And the search result of user "Alice" should not contain these entries:
+      | file1.txt |
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
+
+
+  Scenario Outline: search restored files through a tag
+    Given using <dav-path-version> DAV path
+    And user "Alice" has uploaded file with content "hello world" to "file1.txt"
+    And user "Alice" has uploaded file with content "Namaste nepal" to "file2.txt"
+    And user "Alice" has created the following tags for file "file1.txt" of the space "Personal":
+      | tag1 |
+    And user "Alice" has deleted file "/file1.txt"
+    When user "Alice" restores the file with original path "/file1.txt" using the trashbin API
```

Comment:
can this be made a `Given` step otherwise we have to check the status code as well.

Reply:
ok